token-swap/program/src/curve/calculator.rs

```diff
@@ -43,16 +43,24 @@ pub struct SwapResult {
     pub new_source_amount: u128,
     /// New amount of destination token
     pub new_destination_amount: u128,
-    /// Amount of source token swapped
+    /// Amount of source token swapped (includes fees)
     pub source_amount_swapped: u128,
     /// Amount of destination token swapped
-    pub amount_swapped: u128,
+    pub destination_amount_swapped: u128,
```

Comment:
it's funny, I had done the same thing in the follow-up work, but wanted to keep the current PR small

Reply:
Ha, I debated too. But felt the readability was worth it :)